examples/README.md

```diff
@@ -0,0 +1,30 @@
+## Installation & deployment tips 
+1. You need to configure your node to utilize GPU. In order this can be done the following way: 
+    * Install [nvidia-docker2](https://github.com/NVIDIA/nvidia-docker)
```

Comment:
> nvidia-docker2

Is it a typo?

Reply:
? It is literally nvidia-docker2.

Reply:
Oh gotcha. You mean https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)